setup.py

```diff
@@ -21,6 +21,7 @@
 setup(
     name='packet_queue',
     version='0.1.0',
+    zip_safe=False,  # https://github.com/ldx/python-iptables/issues/76
```

Comment:
I don't think the explanatory URL is necessary; it's common to have non-zip-safe dependencies, isn't it? Just a comment that this is because of the iptables module should be enough.